boards/adafruit/metro_esp32s3/adafruit_metro_esp32s3_appcpu.dts

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright (c) 2024 Espressif Systems (Shanghai) Co., Ltd.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+/dts-v1/;
+
+#include <espressif/esp32s3/esp32s3_appcpu.dtsi>
+/ {
+    model = "Adafruit Metro ESP32S3 APPCPU";
```

Comment:
tab indent for dts files

Reply:
Thanks for the review @nordicjm!
I tried to fix the indentation/make it consistent across files after your comments yesterday, so if there's a filetype dependency on the style guide I'd appreciate if you can direct me to a resource where I can read up on these differences. I don't quite understand what I'm supposed to change here.

Reply:
Not on the documentation but you have used spaces in these files, devicetree files do not use spaces in zephyr, they need to use tabs

Reply:
What about the dtsi files I edited yesterday? The problems you commented on there were caused by the tab size on github being configured to the equivalent of 8 spaces instead of the 4 spaces I used in my editor. With 4 spaces per tab they looked good, but I changed to spaces for consistency. Should I revert the whole commit, or just for .dts files specifically?

Reply:
There should be no spaces at all, tabs and tabs only

Reply:
Ok. Fixed